Homework2/src/problem3/Board.java

```diff
@@ -0,0 +1,18 @@
+package problem3;
+
+import java.util.ArrayList;
+
+public class Board {
+    ArrayList<Message> messages = new ArrayList<>();
```

Comment:
e de preferat sa folosesti interfata unde e posibil, de ex aici: List<Message> messages = new ArrayList<>(); pentru ca pe viitor, daca schimbi implementarea(inlocuiesti ArrayList cu alt tip de lista) sa fie foarte usor de facut aceasta schimbare si sa nu fie  nevoie sa te mai atingi de alte bucati de cod; uite si aici o explicatie: https://stackoverflow.com/questions/147468/why-should-the-interface-for-a-java-class-be-preferred